setup.py

```diff
@@ -30,18 +30,15 @@ def read(fname):
     "jobwatcher = jobwatcher.jobwatcher:main",
 ]
 version = "2.4.0"
-requires = ["boto3>=1.7.55", "python-dateutil>=2.6.1", "retrying>=1.3.3", "future>=0.17.1", "configparser>=3.7.4"]
-
-if sys.version_info[:2] == (2, 6):
-    # For python2.6 we have to require argparse since it
-    # was not in stdlib until 2.7.
-    requires.append("argparse>=1.4")
-    requires.append("idna==2.6")
-    requires.append("paramiko==2.3.3")
-    requires.append("cryptography==2.1.4")  # dependency of paramiko
-    requires.append("pycparser==2.18")
-else:
-    requires.append("paramiko>=2.4.2")
+requires = [
+    "requests>=2.21.0",
+    "boto3>=1.7.55",
+    "python-dateutil>=2.6.1",
+    "retrying>=1.3.3",
+    "future>=0.17.1",
+    "configparser>=3.7.4",
+    "paramiko>=2.4.2",
+]
 
 setup(
```

Comment:
shall we add `python_requires='>=3.4'`?

https://packaging.python.org/guides/dropping-older-python-versions/

Reply:
added